test/cbmc/proofs/DNS/DNSgetHostByName/DNSgetHostByName_harness.c

```diff
@@ -22,15 +22,17 @@
 uint32_t FreeRTOS_dnslookup( const char * pcHostName );
 Socket_t DNS_CreateSocket( TickType_t uxReadTimeout_ticks );
 void DNS_CloseSocket( Socket_t xDNSSocket );
-void DNS_ReadReply( Socket_t xDNSSocket,
+BaseType_t DNS_ReadReply( const ConstSocket_t xDNSSocket,
```

Comment:
Do we need `const ConstSocket_t`? Or should `ConstSocket_t` be enough?

Reply:
fixed